frontend/src/components/MeetingRoom/EmojiGrid/EmojiGrid.tsx

```diff
@@ -14,28 +15,57 @@ import displayOnDesktop from '../../../utils/displayOnDesktop';
  * @returns {ReactElement} - The EmojiGrid Component.
  */
 const EmojiGrid = (): ReactElement => {
+  const { openEmojiGrid, setOpenEmojiGrid } = useSessionContext();
   const anchorRef = useRef<HTMLButtonElement>(null);
-  const [open, setOpen] = useState<boolean>(false);
+  const isSmallViewport = useIsSmallViewport();
+  const [open, setOpen] = useState<boolean>(isSmallViewport ? openEmojiGrid : false);
```

Comment:
Do we need both states? I think one is enough, you seem to update both at the same time

Reply:
Yes, we do! If we toggle the open state through the emoji button, we save whether it's open or not as a user preference for the next time we use the `HiddenToolbarItems` drawer. If we use a click/tap away, we don't save the user's preference to open/close the emojis, but we do update its open state.

Reply:
But can you not just use the higher up state and remove the local state?

Reply:
Sorry, I might not be clear or I might not be following. To my understanding, we need:
1. a slice of state to track whether the emoji grid will be opened on the `PopupMenuToggleButton` being toggled
2. a slice of state to track whether the emoji grid is opened or closed by the `EmojiGrid` button

We can sync up on this, too.

Reply:
Aaahhh, you're right. Thanks!